### DISCUSSION TITLE
Are grouped song scenes in OBS switched via auto switch function?

### DISCUSSION THREAD

**Discussion by kozaka-tv** [Th0rstenf/Rocksniffer-to-Streamer.bot-extension#110] [General]
It must be clarified, that scenes grouped in OBS or nested are switched with the auto switch function or not. Probably yes, but this use case must be tested.

_Originally posted by @kozaka-tv in https://github.com/Th0rstenf/Rocksniffer-to-Streamer.bot-extension/discussions/103#discussioncomment-5608652_

**Comment by Th0rstenf** [comment]
I'm not sure what you mean with grouped. Could you give a specific example?

**Comment by kozaka-tv** [comment]
In OBS I mean. I have multiple scenes in beside each other. But you mentioned, that others having only one Group os SongScenes in OBS. So the question is just, that this use case is also working with the auto switch function. That grouped scenes will be switched or not. Or nested scenes. But I will just test it out I think. I wonder just that this will work or not.

**Comment by kozaka-tv** [comment]
Like this:

![image](https://user-images.githubusercontent.com/4964462/232325821-1f326b89-0bed-4062-8124-b8af34c9dba4.png)

But...how do you change the scenes? Or die I understand wrong? How do you switch scenes then? Change the visibility? But then our auto switch will not switch, or better said, the songScenes variable will be only: RS (according to my example). And this is just shown, and the switch happens differently, except the user has 2 such scenes as in the example, and then songScenes = RS, RSAnother. And only this 2 will be switched... :) I am lost :)

**Reply by Th0rstenf** [reply]
You misunderstood. You have nested scenes as as a few of them contain elements you want to have all the time, and then you work with visibility / (move) filters with other.

**Reply by kozaka-tv** [reply]
Ah ok, then the auto switch is working only, if somebody has multiple scenes as I have. Then this is not a question, as that visibility and filter changes must be made by the user in OBS somehow.
